extide/gradle/src/org/netbeans/modules/gradle/spi/actions/ProjectActionMappingProvider.java

```diff
@@ -17,13 +17,14 @@
  * under the License.
  */
 
-package org.netbeans.modules.gradle.actions;
+package org.netbeans.modules.gradle.spi.actions;
 
 import java.util.Set;
 import org.netbeans.modules.gradle.api.execute.ActionMapping;
 
 /**
```

Comment:
Consider adding better Javadoc. Description/example of "where/how to register this interface" was something I'd have expected here.